kura/manifest_pom.xml

```diff
@@ -137,6 +124,55 @@
 				<module>org.eclipse.kura.protocol.can.test</module>
 			</modules>
 		</profile>
+		
+		<profile>
+			<id>test-repository</id>
+			<activation>
+				<property>
+					<name>useTestRepo</name>
+					<value>true</value>
+				</property>
+				<activeByDefault>false</activeByDefault>
+			</activation>
+			<properties>
+			</properties>
+			<distributionManagement>
+				<repository>
+					<id>releases</id>
+					<name>releases</name>
+					<url>http://172.17.0.2:8081/content/repositories/releases</url>
```

Comment:
I don't think it is a good idea to use internal addressing in an open source project. I would recommend to use something like maven settings for that.